diff --git a/core/lib/Drupal/Core/PathProcessor/PathProcessorFront.php b/core/lib/Drupal/Core/PathProcessor/PathProcessorFront.php
index 161661b..2b9da15 100644
--- a/core/lib/Drupal/Core/PathProcessor/PathProcessorFront.php
+++ b/core/lib/Drupal/Core/PathProcessor/PathProcessorFront.php
@@ -39,7 +39,8 @@ public function __construct(ConfigFactoryInterface $config) {
    * Implements Drupal\Core\PathProcessor\InboundPathProcessorInterface::processInbound().
    */
   public function processInbound($path, Request $request) {
-    if ($path === '/') {
+    $front_page = $this->config->get('system.site')->get('page.front');
+    if ($path === $front_page || $path === "/") {
       $path = $this->config->get('system.site')->get('page.front');
     }
     return $path;
