diff --git a/core/lib/Drupal/Core/Mail/MailManager.php b/core/lib/Drupal/Core/Mail/MailManager.php
index 663cfd4..dd14078 100644
--- a/core/lib/Drupal/Core/Mail/MailManager.php
+++ b/core/lib/Drupal/Core/Mail/MailManager.php
@@ -238,7 +238,7 @@ public function mail($module, $key, $to, $langcode, $params = array(), $reply =
             '%to' => $message['to'],
             '%reply' => $message['reply-to'] ? $message['reply-to'] : $this->t('not set'),
           ));
-          drupal_set_message($this->t('Unable to send email. Contact the site administrator if the problem persists.'), 'error');
+          //drupal_set_message($this->t('Unable to send email. Contact the site administrator if the problem persists.'), 'error');
         }
       }
     }
